### DISCUSSION TITLE
Is deepgenx online yet?

### DISCUSSION THREAD

**Discussion by krazykoder** [DeepGenX/CodeGenX#25] [Q&A] [answered]
https://www.deepgenx.com/signin
Website shows that it is still being migrated to better hardware. Wondering when this will be available again

**Comment by Aryagm** [comment]
Hi!

We ran into some compatibility issues with the new hardware.

You can expect CodeGenX to be online again by the end of this week.

Sorry for the inconvenience.

Thanks and Regards,
Arya Manjaramkar

**Reply by KongNan** [reply]
Best wishes ❤️

**Comment by Aryagm** [comment, marked as answer]
> https://www.deepgenx.com/signin Website shows that it is still being migrated to better hardware. Wondering when this will be available again

Hi,

CodeGenX is finally online again! 🎉

If you are an existing user you will need to update the extension in VsCode. If you are new you can get your API token on our [website](https://deepgenx.com/).

Sorry for the long wait.

Thanks and Regards,
Arya Manjaramkar

**Reply by KongNan** [reply]
Good news ❤️ 

but I can't receive API key , please take a look?

**Reply by Aryagm** [reply]
Hi,

The problem should be fixed now!

Sorry for the inconvenience,

Thanks and Regards,
Arya Manjaramkar

**Reply by KongNan** [reply]
Thank you , but no email :(

**Reply by Aryagm** [reply]
Hmm, that is very odd. Let me take another look. I'll hopefully get back to you tomorrow with a solution.

Sorry again for the inconveniences.

Thanks and Regards,
Arya Manjaramkar

**Reply by Ashutosh-Srivastav** [reply]
Hi @Aryagm , By any chance is the issue with email sending resolved? Thanks!

**Reply by tereshchuk-dev** [reply]
Similar problem. No confirmation email :(
